config/settings.yml

```diff
@@ -140,7 +140,7 @@ fossgis_valhalla_url: "https://valhalla1.openstreetmap.de/route"
 wikidata_api_url: "https://www.wikidata.org/w/api.php"
 wikimedia_commons_url: "https://commons.wikimedia.org/wiki/"
 # URL to donation form
-donation_url: "https://supporting.openstreetmap.org/"
+donation_url: "https://supporting.openstreetmap.org/donate/"
```

Comment:
I changed the link to go directly to [the donation form](https://supporting.openstreetmap.org/donate/) instead of [the landing page](https://supporting.openstreetmap.org/). I think this is a better match for the user’s intent when they click the Donate button in the navigation bar. In general, any extraneous step standing in the way of a signup or donation flow will tend to lose a certain share of users, which we don’t want. The donation form also mentions opportunities to become a member, but making a donation is the focus.